paddle/framework/op_registry.h

```diff
@@ -358,30 +357,20 @@ class OpKernelRegistrar : public Registrar {
 /**
  * Macro to register Operator.
  */
-#define REGISTER_OP(op_type, op_class, op_maker_class)                        \
+#define REGISTER_OP(op_type, op_class, op_maker_class, grad_op_type,          \
+                    grad_op_class)                                            \
```

Comment:
when we change the registration style, we need to discuss a previous problem. The user's code needs to write `USE_OP(OP)`, `USE_OP(OP_GRAD)` twice.  It is annoying.